app/scripts/views/base.js

```diff
@@ -552,7 +552,8 @@ define(function (require, exports, module) {
       err.logged = true;
 
       if (typeof console !== 'undefined' && console) {
-        console.error(err.message || err);
+        var errorMessage = err.errorModule ? err.errorModule.toInterpolatedMessage(err) : (err.message || err);
+        console.error(errorMessage);
```

Comment:
Sorry for confusion, I just meant I saw some test cases where it checked that a sentry error was logged. I think you are correct, `sinon` can spy console.